esp/src/eclwatch/HPCCPlatformWidget.js

```diff
@@ -234,6 +235,22 @@ define([
                 topic.subscribe("hpcc/monitoring_component_update", function (topic) {
                     context.checkMonitoring(topic.status);
                 });
+                this.storage = new ESPUtil.LocalStorage();
+                this.storage.start();
+                this.storage.on("storageUpdate", function(event) {
+                    context._onUpdateFromStorage(event)
+                });
+            },
+
+            _onUpdateFromStorage: function (value){
```

Comment:
Minor naming:  Your using "event" and then "value" which contains an event - maybe they should both be "msg"?